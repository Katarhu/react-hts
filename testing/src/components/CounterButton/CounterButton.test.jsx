import React from "react";
import renderer from "react-test-renderer";

import { CounterButton } from "./CounterButton";

test('should match snapshot', () => {
  const button = renderer.create(<CounterButton>+</CounterButton>);

  expect(button).toMatchSnapshot();
});
