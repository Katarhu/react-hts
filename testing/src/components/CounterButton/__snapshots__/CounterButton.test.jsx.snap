// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<button
  className="clicker-button"
  type="button"
>
  +
</button>
`;
